network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/config/NetPDefaultConfigProvider.kt

```diff
@@ -48,8 +43,28 @@ data class PcapConfig(val filename: String, val snapLen: Int, val fileSize: Int)
 @ContributesBinding(VpnScope::class)
 class RealNetPDefaultConfigProvider @Inject constructor(
     private val netPExclusionListRepository: NetPExclusionListRepository,
+    private val dispatcherProvider: DispatcherProvider,
+    private val netPSettingsLocalConfig: NetPSettingsLocalConfig,
 ) : NetPDefaultConfigProvider {
     override fun exclusionList(): Set<String> {
         return netPExclusionListRepository.getExcludedAppPackages().toSet()
     }
+
+    override suspend fun routes(): Map<String, Int> = withContext(dispatcherProvider.io()) {
+        val defaultImpl = object : NetPDefaultConfigProvider {}
```

Comment:
NIT: this is not used anymore

Reply:
damn, I was took quick merging, we'll remove it in another PR. Sorry about that